src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <section>
+    <div className="calculator">
+      <div className="group-one">
+        <Button name="AC" />
+        <Button name="+/-" />
+        <Button name="%" />
+        <Button name="÷" />
+      </div>
+      <div className="group-two">
+        <Button name="7" />
+        <Button name="8" />
+        <Button name="9" />
+        <Button name="X" />
+      </div>
+      <div className="group-three">
+        <Button name="4" />
+        <Button name="5" />
+        <Button name="6" />
+        <Button name="-" />
+      </div>
+      <div className="group-four">
+        <Button name="1" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.